.github/workflows/format_push.yml

```diff
@@ -1,5 +1,9 @@
 name: Lint Format
 
+permissions:
+  contents: write
+  pull-requests: write
```

Comment:
Doesnt run on pulls?
```suggestion
```

Reply:
It creates a new PR, so I was under the expectation that it needed write access.

Reply:
Though it uses the bot token, so perhaps not. I'll remove it, if it breaks we can put it back in I suppose.